docker/ci/requirements.txt

```diff
@@ -15,7 +15,6 @@ pyfakefs==3.4.3
 pylint==1.9.3
 python-mimeparse==1.5.2
 PyYAML==5.1
```

Comment:
nit: do we need PyYAML in both places, here and in src/requirements.txt ?

Reply:
I think this is needed for lint run
src/local/butler/lint.py:import yaml
It was breaking things, that is why explicitly got added in https://github.com/google/clusterfuzz/pull/336

Reply:
Found it, yes butler lint runs before setup(setup runs butler bootstrap). and lint needs pyyaml
https://github.com/google/clusterfuzz/blob/master/cloudbuild.yaml#L31

Reply:
Got it, thanks!